targets/xorg

```diff
@@ -189,18 +188,18 @@ else
 fi
 
 # Install xorg, no video drivers
-if [ -z "$inteldriver" -a -n "$backport" ] && release -eq precise; then
+# if [ -z "$inteldriver" -a -n "$backport" ] && release -eq precise; then
     # xorg is packaged wrong on ARM and conflicts with xserver-xorg-lts-quantal
     # so replace it with something not broken
-    install_dummy xorg -- xserver-xorg$backport libgl1-mesa-glx$backport \
-        libgl1-mesa-dri$backport libglu1-mesa xfonts-base x11-apps \
-        x11-session-utils x11-utils x11-xfs-utils x11-xkb-utils \
-        x11-xserver-utils xauth xinit xfonts-utils xkb-data xorg-docs-core \
-        xterm x11-common xinput
-else
-    install xorg $installvideodrivers -- xserver-xorg-video-all$backport \
+#     install_dummy xorg -- xserver-xorg$backport libgl1-mesa-glx$backport \
+#         libgl1-mesa-dri$backport libglu1-mesa xfonts-base x11-apps \
+#         x11-session-utils x11-utils x11-xfs-utils x11-xkb-utils \
+#         x11-xserver-utils xauth xinit xfonts-utils xkb-data xorg-docs-core \
+#         xterm x11-common xinput
+# else
+    install xorg $installvideodrivers \
```

Comment:
the backports logic only affected Ubuntu 12.04.  I guess we could remove it since that release is long gone.

The only thing that actually affects the stuff you're installing is this change here, which makes it so that now xorg can bring in all of the video drivers, even the ones we don't want.

I think what you actually want to do is determine the video driver you need for your system and add to the logic that sets `installvideodrivers` up above (around line 80).